pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -3703,3 +4154,14 @@ func TestGetIndexingReferencesToArrayParams(t *testing.T) {
 		})
 	}
 }
+
+func withAlphaAPIContext(ctx context.Context) context.Context {
```

Comment:
We have helper function to set API fields and other feature flags:

pkg/apis/config/testing/featureflags.go

Reply:
Thanks @QuanZhang-William Deleted!